eatngo-auth-common/src/main/kotlin/com/eatngo/auth/token/TokenController.kt

```
@@ -0,0 +1,44 @@
+            response.addCookie(
+                Cookie("access_token", newAccessToken).apply {
+                    isHttpOnly = true
+                    secure = true
+                    path = "/"
+                    maxAge = 600
+                }
+            )
+
+            ResponseEntity.ok("Access token refreshed") // TODO 응답 포맷 변경
+
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)